wiki/People/osu!_Alumni/bg.md

```diff
@@ -1,6 +1,6 @@
 ---
+outdated_since: 52a908a6fa4efa0ee1a968563c88a5eeeff89f62
 outdated_translation: true
-outdated_since: 3fe983b3a55970aed91a0bcdca028253d918ae5d
```

Comment:
these should never be updated unless the previous commit hash was erroneous, and i don't see the reason mentioned in the pr desc